DESCRIPTION

```diff
@@ -28,7 +28,8 @@ Imports:
     tibble (>= 1.4.2),
     tools,
     withr (>= 1.0.0),
-    xfun (>= 0.1)
+    xfun (>= 0.1),
+    future.apply
```

Comment:
For the example, we'd also need to declare `future` as an import explicitly (although it will be available anyways since we import `future.apply`)